src/link_rfu_2.c

```diff
@@ -114,79 +179,83 @@ const char sUnref_082ED6E8[][15] = {
     "RFU SEND ERR",
     "RFU CP POLL"
 };
-const char sUnref_082ED6E9[][16] = {
+
+static const char sASCII_RecoverCmds[][16] = {
     "              ",
     "RECOVER START ",
     "DISSCONECT    ",
     "RECOVER SUUSES",
     "RECOVER FAILED"
 };
-const TaskFunc gUnknown_082ED7E0[] = {
+
+static const TaskFunc sUnknown_082ED7E0[] = {
     sub_801084C,
-    sub_8010AAC,
+    Task_ExchangeLinkPlayers,
     sub_8010D0C
 };
-const char gUnknown_082ED7EC[] = "PokemonSioInfo";
-const char gUnknown_082ED7FC[] = "LINK LOSS DISCONNECT!";
-const char gUnknown_082ED814[] = "LINK LOSS RECOVERY NOW";
-
-ALIGNED(4) const char gUnknown_082ED82C[31] = {' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',0x00};
-const char gUnknown_082ED84B[16] = {' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',' ',0x00};
-const char gUnknown_082ED85B[9] = {' ',' ',' ',' ',' ',' ',' ',' ',0x00};
-ALIGNED(4) const char gUnknown_082ED864[2] = {' ',0x00};
-const char gUnknown_082ED866[2] = {'*',0x00};
-const char gUnknown_082ED868[8] = "NOWSLOT";
-const char gUnknown_082ED870[12] = "           ";
-const char gUnknown_082ED87C[12] = "CLOCK DRIFT";
-const char gUnknown_082ED888[12] = "BUSY SEND  ";
-const char gUnknown_082ED894[12] = "CMD REJECT ";
-const char gUnknown_082ED8A0[12] = "CLOCK SLAVE";
-const char gUnknown_082ED8A8[3][8] = {
+
+static const char sASCII_PokemonSioInfo[] = "PokemonSioInfo";
```

Comment:
Some of these string constants can be inlined.

Reply:
Why should we? pokefirered does this, but `sASCII_PokemonSioInfo` for example is used more than once. Seems just as reasonable to collect all these debug strings in one place rather than scatter them

Reply:
iirc agbcc does optimize the assembly of multiply-referenced C strings

Reply:
I don't see a reason we should make that change